### PR TITLE
Improving controller tests

### PR DESCRIPTION
**Pull request by escowles**
* Adding show routes for embargo/lease so tests work
* Improving pending messages for characterization tests

### CODE REVIEW

**Comment by jcoyne** [comment]
I don't see where the tests are calling the Embargo or Lease show routes.  I don't think that controller provides a show route.

**Comment by jcoyne** [comment]
I think the access controls are failing and it's stupidly redirecting to show, even though there is no show page.

**Comment by escowles** [comment]
Yes, the access controls are stupidly redirecting to non-existent show, which results in a routing error.  Adding a show route (even with no action or views) makes the tests pass.  Is there a way to make the access control failure redirect to #index instead?

**Comment by jcoyne** [comment]
I think we need something like this: https://github.com/projecthydra/sufia/commit/2101779801d9dd7700949bfce9fdd1faeca2ddd0

**Comment by escowles** [comment]
@jcoyne I've updated the PR to override deny_access to handle the errors, instead of defining bogus routes.

**Comment by jcoyne** [comment]
@escowles Thanks, much better now.

File: spec/controllers/curation_concerns/generic_files_controller_spec.rb [hunk 1]
```diff
@@ -72,7 +72,8 @@
       end
 
       context "when the file has a virus" do
-        xit "displays a flash error" do
+        it "displays a flash error" do
+          skip "pending hydra-works#89"
```
Comment (jpstroop) [inline spec/controllers/curation_concerns/generic_files_controller_spec.rb]: :+1: to annotating why a spec is being skipped. This was a big headache in the initial code shredding project.

**Pull request merged**